integration-libs/cdc/manage-users/manage-users-button/cdc-list.module.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { NgSelectModule } from '@ng-select/ng-select';
+import { ConfigModule, I18nModule, UrlModule } from '@spartacus/core';
+import {
+  IconModule,
+  KeyboardFocusModule,
+  PaginationModule,
+  SplitViewModule,
+  TableModule,
+} from '@spartacus/storefront';
+import { CdcListComponent } from './cdc-list.component';
+import { PopoverModule } from '@spartacus/storefront';
+import {
+  ListModule,
+  MessageModule,
+} from '@spartacus/organization/administration/components';
+import { cdcUserCmsConfig } from '../cdc-user.config';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    RouterModule,
+    SplitViewModule,
+    TableModule,
+    IconModule,
+    UrlModule,
+    I18nModule,
+    PaginationModule,
+    NgSelectModule,
+    FormsModule,
+    MessageModule,
+    KeyboardFocusModule,
+    PopoverModule,
+    ConfigModule.withConfig(cdcUserCmsConfig),
```

Comment:
The file `integration-libs/cdc/manage-users/cdc-user.config.ts` is not needed.

Instead use - 

```suggestion
    ],
    providers: [
      provideDefaultConfig(<CmsConfig>{
        cmsComponents: {
          ManageUsersListComponent: {
            component: CdcListComponent,
          },
        },
      }),
    ],
```

Reply:
done